src/dmd/expressionsem.d

```diff
@@ -6287,6 +6287,42 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
         result = e.expressionSemantic(sc);
     }
 
+    /*
+     * Get the expression initializer for a specific struct
+     *
+     * Params:
+     *  sd = the struct for which the expression initializer is needed
+     *  loc = the location of the initializer
+     *  sc = the scope where the expression is located
+     *  t1 = the type of the expression
+     *
+     * Returns:
+     *  The expression initializer
+     */
```

Comment:
If we want to be strict about the style (https://dlang.org/dstyle.html#phobos_documentation) we should not prepend `*` characters on each line.  But I don't think it should hold up this PR.

Reply:
The remark about not using leading `*` in documentation was added to the style guide in January of this year and reflects Phobos but doesn't reflect the style of the existing DMD codebase. IMO the most important thing is maintaining consistency within a document.